usr/lib/systemd/system/greenboot-service-monitor.service

```
@@ -0,0 +1,22 @@
+#  SPDX-License-Identifier: LGPL-2.1+
+#
+#  This file is part of greenboot.
+#
+#  greenboot is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Monitor user services
+Before=boot-complete.target
+OnFailure=redboot.target
+After=network.target sshd.service
```

Could you pull in the change from https://github.com/fedora-iot/greenboot/pull/72 and try this?
```suggestion
After=default.target graphical.target multi-user.target
```

---

do this causing `systemd: Freezing execution ` on reboot, similar to when trying to enable system-boot-check-no-failure.service, not sure if its my system or things in general.

---

@LorbusChris could you please test your suggested method and let me know?

---

I don't see the commit from #72 in this PR. Did you try to test this together with that change? It is crucial that is added here. You can simply cherry-pick it on top of this branch. Please provide full output logs of any failures, I don't have cycles to test this.